Models/AbstractProjectModels/AbstractDefaultModel.php

```diff
@@ -113,4 +109,9 @@ protected function getLogger(): Logger
 
         return $this->logger;
     }
+
+    public function getSessModel() :AbstractSessionModel
```

Comment:
why didn't you write getSessionModel? your method name contains 3 symbol less, does it make a lot of sense?